### PR TITLE
add profiling docs

### PR DESCRIPTION
**Pull request by szuecs**
Signed-off-by: Sandor Szücs <sandor.szuecs@zalando.de>

### CODE REVIEW

File: docs/operation/operation.md [hunk 1]
```diff
@@ -879,3 +879,55 @@ predicates and filters involved in the route processing:
...
+To enable profiling in skipper you have to use `-enable-profile`. This
+will start a profiling route at `/debug/pprof/profile` on the support
+listener, which defaults to `:9911`.
+
+### Profiling example
+
+Start skipper with enabled profiling:
+
+```
+./bin/skipper -routes-file examples/shadow-2-routes.eskip -enable-profile
```
Comment (AlexanderYastrebov) [inline docs/operation/operation.md:897]: It would be nice to use an existing file (e.g from `./skptesting/`) so that users may actually try it out

Comment (szuecs) [reply docs/operation/operation.md:897]: Before I thought to just use inline route, which does not depend on any CWD, but I missed to replace it :D

**Comment by szuecs** [comment]
:+1:

**Comment by aryszka** [comment]
:+1:

**Pull request merged**